pyenvisalink/dsc_client.py

```diff
@@ -177,6 +190,10 @@ def handle_partition_state_change(self, code, data):
                     lastDisarmedBy = {'last_disarmed_by_user': int(data[1:5])}
                     self._alarmPanel.alarm_state['partition'][partitionNumber]['status'].update(lastDisarmedBy)
 
+                if code == '655':
```

Comment:
Can we add this to the "handle_partition_state_change" event instead? Like for example this can be added to this "if" statement here: https://github.com/Cinntax/pyenvisalink/blob/8c79fdafaaae59a208f99000005e8a3b72ca5c59/pyenvisalink/dsc_client.py#L156